packages/commerce-sdk-react/src/hooks/ShopperCustomers/mutation.test.tsx

```diff
@@ -129,138 +119,126 @@ const CustomerMutationComponent = ({action}: CustomerMutationComponentParams) =>
     )
 }
 
-const tests = (Object.keys(testActionsArgs) as ShopperCustomersMutationType[]).map((key) => {
-    return {
-        hook: key,
-        cases: [
-            {
-                name: 'success',
-                assertions: async () => {
-                    mockAuthCalls()
-
-                    nock('http://localhost:3000')
-                        .patch((uri) => {
-                            return uri.includes('/customer/shopper-customers/')
-                        })
-                        .reply(200, {})
-                    nock('http://localhost:3000')
-                        .put((uri) => {
-                            return uri.includes('/customer/shopper-customers/')
+const tests = (Object.keys(mutationPayloads) as ShopperCustomersMutationType[]).map(
+    (mutationName) => {
+        return {
+            hook: mutationName,
+            cases: [
+                {
+                    name: 'success',
+                    assertions: async () => {
+                        nock(DEFAULT_TEST_HOST)
+                            .patch((uri) => {
+                                return uri.includes('/customer/shopper-customers/')
+                            })
+                            .reply(200, {})
+                            .put((uri) => {
+                                return uri.includes('/customer/shopper-customers/')
+                            })
+                            .reply(200, {})
+                            .post((uri) => {
+                                return uri.includes('/customer/shopper-customers/')
+                            })
+                            .reply(200, {})
+                            .delete((uri) => {
+                                return uri.includes('/customer/shopper-customers/')
+                            })
+                            .reply(204, {})
+
+                        const queryClient = createQueryClient()
+
+                        renderWithProviders(<CustomerMutationComponent mutation={mutationName} />, {
```

Comment:
tried to use `renderHookWithProviders` but ran into some issues, i'll try it again today.